From 2463a427d2f4af2b01dbb83ea52d7eab099be569 Mon Sep 17 00:00:00 2001
From: Tomas Glozar <tglozar@gmail.com>
Date: Fri, 20 Mar 2026 20:06:39 +0100
Subject: [PATCH 3/8] ia64: Add mseal() syscall

mseal() was added after ia64 was removed from the kernel.

Fixes: ff388fe5c481 ("mseal: wire up mseal syscall")
Signed-off-by: Tomas Glozar <tglozar@gmail.com>
---
 arch/ia64/kernel/syscalls/syscall.tbl | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/ia64/kernel/syscalls/syscall.tbl b/arch/ia64/kernel/syscalls/syscall.tbl
index 636854b52b8a..f3e0ece43d34 100644
--- a/arch/ia64/kernel/syscalls/syscall.tbl
+++ b/arch/ia64/kernel/syscalls/syscall.tbl
@@ -382,3 +382,4 @@
 459	common	lsm_get_self_attr		sys_lsm_get_self_attr
 460	common	lsm_set_self_attr		sys_lsm_set_self_attr
 461	common	lsm_list_modules		sys_lsm_list_modules
+462	common	mseal				sys_mseal
-- 
2.52.0

